test/neural_net/style_transfer_testing.hpp

```diff
@@ -0,0 +1,63 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef __TEST_NEURAL_NET_STYLE_TRANSFER_TESTING_HPP
+#define __TEST_NEURAL_NET_STYLE_TRANSFER_TESTING_HPP
+
+#include <functional>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace neural_net_test {
```

Comment:
It will almost certainly be most convenient to use whatever namespace the code under test is in. In this case, `turi::neural_net`.

Reply:
Alright!